ossec_security/README.md

```diff
@@ -0,0 +1,204 @@
+    logs_enabled: true
+    ```
+2. Add this configuration block to your `ossec_security.d/conf.yaml` file to start collecting your logs.
+
+    We will be using the UDP method to collect the OSSEC alerts data.
+    See the [sample ossec_security.d/conf.yaml][8] for available configuration options.
+
+    ```yaml
+      logs:
+      - type: udp
+        port: <PORT>
+        source: ossec-security
+        service: ossec-security
+    ```
+    **Note**: It is recommended not to change the service and source values as these parameters are integral to the pipeline's operation.
+
+3. [Restart the Agent][3].
+
+#### Configure Syslog Message Forwarding from OSSEC.
+  1. Add following configuration in `/var/ossec/etc/ossec.conf`.
+
+      In this example all alerts are sent to 1.1.1.1 on port 8080 in **JSON** format.
+      ```xml
+        <syslog_output>
+          <server>1.1.1.1</server>
+          <port>8080</port>
+          <format>json</format>
+        </syslog_output>
+      ```
+
+      *IP_ADDRESS(1.1.1.1): ip address where your datadog-agent is running.
+
+      *PORT(8080): port on which your datadog-agent is listening.
+
+      Note: Using JSON format is required since OSSEC Security pipeline parses JSON formatted logs only.
+
+  2. After this change is made, the client-syslog process should be enabled:
+      ```shell
+      /var/ossec/bin/ossec-control enable client-syslog
+      ```
+
+  3. Restart OSSEC service:
+      ```shell
+      /var/ossec/bin/ossec-control restart
+      ```
+
+#### Firewall Logs Collection[Optional]:
+Since OSSEC server does not forward the firewall alert logs via syslog by default, if you want to forward firewall alert logs via OSSEC server follow the below step.
+
+  1. Locate `firewall_rules.xml` file.
+  ```
+  /var/ossec/rules/firewall_rules.xml
+  ```
+
+  2. Edit firewall_rules.xml and remove all the occurrences of below line from the file:
+  ```xml
+  <options>no_log</options>
+  ``` 
+
+  3. Restart your OSSEC server:
+  ```shell
+  /var/ossec/bin/ossec-control restart
+  ```
+
+#### Changes in OSSEC Security Datadog log pipeline for timezone other than UTC
+
+Since Datadog expects all the logs in UTC timezone by default, If the timezone of your OSSEC logs is other than UTC, please specify it in the OSSEC Security datadog pipeline.
+
+In order to change the timezone in OSSEC Security pipeline follow the below steps:
+
+  1. Navigate to Pipelines in the datadog cloud account. 
+      - Go to Logs -> Log Stream -> Pipelines.
+
+  2. Search for `OSSEC Security` using Filter Pipelines.
+
+  3. Clone `OSSEC Security` pipeline in order to change the timezone.
+      - Hover over the `OSSEC Security` pipeline and click on the `clone`  button. This will clone the `OSSEC Security` pipeline which will be editable.
+
+  4. Edit the Grok Parser using below steps:
+      - Find a processor with the name "Grok Parser: Parsing OSSEC alerts" and click on the `Edit` button by hovering over the pipeline.
```

Comment:
```suggestion
      - In the cloned pipeline, find a processor with the name "Grok Parser: Parsing OSSEC alerts" and click on the `Edit` button by hovering over the pipeline.
```
If "in the cloned pipeline" is incorrect, provide some other location hint here.

Reply:
Updated as per suggestion